### PR TITLE
Qcad 3.25.2.0

### PR DESCRIPTION
**Pull request by yvesf**
#### Motivation for this change

- update to 3.25.2.0
- fix problem with loading shared library for sqlite of the qcad drawing library

Regarding the shared library: I don't fully understand the problem here. Suggestions welcome.

- [x] Tested using sandboxing ([nix.useSandbox](https://nixos.org/nixos/manual/options.html#opt-nix.useSandbox) on NixOS, or option `sandbox` in [`nix.conf`](https://nixos.org/nix/manual/#sec-conf-file) on non-NixOS linux)
- Built on platform(s)
   - [x] NixOS
- [x] Tested execution of all binary files (usually in `./result/bin/`)

### CODE REVIEW

File: pkgs/applications/misc/qcad/default.nix [hunk 1]
```diff
@@ -61,6 +61,9 @@ mkDerivationWith stdenv.mkDerivation rec {
     cp -r plugins $out/lib/plugins
     cp -r patterns $out/lib/patterns
 
+    # workaround to fix the library browser:
+    cp -r ${qt5.qtbase}/${qt5.qtbase.qtPluginPrefix}/sqldrivers $out/lib/plugins
```
Comment (symphorien) [inline pkgs/applications/misc/qcad/default.nix:65]: How about making a symlink, to save some disk space?

Comment (yvesf) [reply pkgs/applications/misc/qcad/default.nix:65]: done

**Comment by symphorien** [comment]
I know this is only an update, but the derivation is not supposed to take qt5 as argument: https://nixos.org/manual/nixpkgs/stable/#qt-default-nix-co-1

**Comment by yvesf** [comment]
> I know this is only an update, but the derivation is not supposed to take qt5 as argument: https://nixos.org/manual/nixpkgs/stable/#qt-default-nix-co-1

thanks for this hint, I made a separate PR for that: https://github.com/NixOS/nixpkgs/pull/98327

**Pull request merged**